compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/ComposeScene.skiko.kt

```diff
@@ -583,13 +583,24 @@ class ComposeScene internal constructor(
     }
 
     private fun processPress(event: PointerInputEvent) {
-        val owner = hoveredOwner(event)
-        if (focusedOwner.isAbove(owner)) {
-            focusedOwner?.onDismissRequest?.invoke()
-        } else {
-            owner?.processPointerInput(event)
-            pressOwner = owner
+        val hoveredOwner = hoveredOwner(event)
+        val aboveOwners = owners
+            .filter { it != hoveredOwner && it.isAbove(hoveredOwner) }
+            .reversed() // Iterate from top to bottom
+        for (owner in aboveOwners) {
+            if (!owner.isHovered(event)) {
+                owner.onClickOutside?.invoke()
+            } else {
+                // Stop once the position of in bounds of the owner
+                break
+            }
+            if (owner == focusedOwner) {
+                // Stop if it's in focus, do not pass the event to hovered owner
+                return
+            }
         }
```

Comment:
We said we'll preserve the existing behaviour, no?

Reply:
> existing behaviour
of android

Current desktop overloads keeps behave the same too because of reusing `focusable` flag in passing `onClickOutside` parameter

Reply:
I think we wanted to preserve the current desktop behaviour, which calls `onClickOutside` only for the focused owner (if it's above the clicked owner).

Even if we do change the behaviour, this is much clearer (also better performance-wise)
```
        for (owner in owners.asReversed()){
            if (owner == focusedOwner)
                return
            if (owner.isHovered(event.pointers.first().position))
                break
            owner.onClickOutside?.invoke()
        }
```

Reply:
No, this code won't work - focused owner should receive onClickOutside. But we can probably drop `filter` call, yes

Reply:
Then 
```
        for (owner in owners.asReversed()){
            if (owner.isHovered(event.pointers.first().position))
                break
            owner.onClickOutside?.invoke()
            if (owner == focusedOwner)
                return
        }
```

Reply:
> calls `onClickOutside` only

No, it's incorrect scenario. Initially designed behaviour for desktop is preserved, but now it also matches android implementation

Reply:
`asReversed` won't work too - we should have a copy to get ready for list modifications

Reply:
Hmm, you're right. But let's at least make it simpler and not O(n^2).